### PR TITLE
docs: update sca nacos-example readme file

### PR DESCRIPTION
**Pull request by yuluo-yx**
### Describe what this PR does / why we need it

- For #3525

### CODE REVIEW

File: spring-cloud-alibaba-examples/nacos-example/readme-zh.md [hunk 1]
```diff
@@ -0,0 +1,610 @@
...
+Spring Boot 应用支持通过 Endpoint 来暴露相关信息，Spring Cloud Alibaba Nacos Config Starter 也支持这一点。
+
+在使用之前需要在 maven 中添加 `spring-boot-starter-actuator`依赖，并在配置中允许 Endpoints 的访问。
+
+* Spring Boot 1.x 中添加配置 management.security.enabled=false
+* Spring Boot 2.x 中添加配置 management.endpoints.web.exposure.include=*
+
+Spring Boot 1.x 可以通过访问 http://127.0.0.1:18084/nacos_config 来查看 Nacos Endpoint 的信息。
+
+Spring Boot 2.x 可以通过访问 http://127.0.0.1:18084/actuator/nacosconfig 来访问。
```
Comment (ruansheng8) [inline spring-cloud-alibaba-examples/nacos-example/readme-zh.md]: 这里只保留 Spring Boot 3.x 即可

File: spring-cloud-alibaba-examples/nacos-example/readme-zh.md [hunk 2]
```diff
@@ -0,0 +1,610 @@
...
+
+2. 在浏览器地址栏中输入 http://127.0.0.1:18083/echo-feign/12345，点击跳转，可以看到浏览器显示 nacos-discovery-provider-example 返回的消息 "hello Nacos Discovery 12345"，证明服务发现生效。
+
+![feign](https://cdn.nlark.com/lark/0/2018/png/54319/1536986311685-6d0c1f9b-a453-4ec3-88ab-f7922d210f65.png)
+
+#### 原理
+
+##### 服务注册
+
+Spring Cloud Alibaba Nacos Discovery 遵循了 spring cloud common 标准，实现了 AutoServiceRegistration、ServiceRegistry、Registration 这三个接口。
```
Comment (ruansheng8) [inline spring-cloud-alibaba-examples/nacos-example/readme-zh.md]: `spring cloud common` 首字母大写

Comment (yuluo-yx) [reply spring-cloud-alibaba-examples/nacos-example/readme-zh.md]: done!

File: spring-cloud-alibaba-examples/nacos-example/readme-zh.md (hunk 1 above)
Comment (yuluo-yx) [reply spring-cloud-alibaba-examples/nacos-example/readme-zh.md]: done!

**Comment by yuluo-yx** [comment]
I'm not sure it's not my code problem that caused ci to fail. pls re-run ci again.  tks. @ruansheng8

**Comment by ruansheng8** [comment]
LGTM.

**Pull request merged**